projector-server/src/main/kotlin/org/jetbrains/projector/server/ProjectorServer.kt

```diff
@@ -541,7 +544,7 @@ class ProjectorServer private constructor(
 
         selectedOption = JOptionPane.showOptionDialog(
           null,
-          "Somebody ($ipString) wants to connect with $accessType access. Allow the connection?",
+          "Somebody (${getHostName(remoteAddress)}) wants to connect with $accessType access. Allow the connection?",
```

Comment:
I get IP here... Testing in my LAN. If you think something can be easily improved, let's do it in this PR.

Reply:
How does look a result of `nslookup` for that IP? Please, run in terminal:
> nslookup x.x.x.x

Reply:
```
servb@UNIT-1259:~$ nslookup 192.168.1.100
** server can't find 100.1.168.192.in-addr.arpa: NXDOMAIN
```

Reply:
There is no record in DNS for this IP. What name did you expect to see?

Reply:
Name of that PC.